packages/core/strapi/lib/commands/utils/formatter.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const { createLogger, createOutputFileConfiguration } = require('@strapi/logger');
+const chalk = require('chalk');
+
+const logger = createLogger(createOutputFileConfiguration('data-transfer.log'));
+
+const errorColors = {
+  fatal: chalk.red,
+  error: chalk.red,
+  silly: chalk.yellow,
+};
+
+const formatDiagnosticErrors = ({ details, kind }) => {
```

Comment:
We're not only formatting errors here, I would suggest renaming this to formatDiagnostic